utils/page-utils.ts

```diff
@@ -585,9 +585,16 @@ export class PageUtils {
      *
      * @return  {Promise<void>}
      */
-    public cleanUp = async (): Promise<void> => {
-        // Remove helper plugin.
-        await uninstallPlugin('wp-rocket force-wp-mobile');
+    public cleanUp = async (plugin = 'wp-rocket'): Promise<void> => {
+        if(plugin === 'wp-rocket') {
+            // Remove helper plugin.
+            await uninstallPlugin('wp-rocket force-wp-mobile');
+        }
+
+        if(plugin === 'backwpup') {
+            // Remove helper plugin.
+            await uninstallPlugin('backwpup-pro backwpup');
+        }
```

Comment:
here too, we should avoid this kind of changes for now as it's too deep into wpr tests, we will refactor these I believe in upcoming sprints when we will split into different repos. for now, it's ok to take out the useful helper functions and use it directly in the hook, like the `uninstallPlugin` function.